components/cdc/src/endpoint.rs

```diff
@@ -306,6 +319,47 @@ impl ResolvedRegionHeap {
         self.clear();
         self.heap.shrink_to(min_capacity);
     }
+
+    // extreme outier match the following two conditions:
+    // 1. https://en.wikipedia.org/wiki/Box_plot
+    // 2. the gap with min_ts is larger than RAW_RESOLVED_TS_OUTLIER_THRESHOLD.
+    fn get_extreme_outlier(&self, min_ts: TimeStamp) -> Vec<u64> {
```

Comment:
I think we can only judge the smallest timestamp, to reduce code complexity. The deadlock is rare.

Reply:
Done, return one maximally. As MinTs task will run periodically, all regions can also be resolved even if more than one regions have dead lock.